user-guide/Standard_Apps/Monitoring_Solutions/PLM_Tool/Tutorial_Creating_PLM_activity.md

```diff
@@ -0,0 +1,148 @@
+---
+uid: Tutorial_Creating_PLM_activity
+---
+
+# Creating a PLM activity
+
+The DataMiner Planned Maintenance tool facilitates the scheduling and management of one-time or recurring maintenance events for the various resources and entities within a system. Through a unified interface, you can define system resources, schedule maintenance events, and observe historical maintenance records. Once you have defined resource types, scheduling maintenance events is only a few clicks away. For this, you will need to specify an event title, a resource and resource type, the start and end time, and the type of recurrence.
+
+This tutorial will show how you can deploy and configure the PLM Solution and then create a PLM activity for a Windows server that is being monitored on a DMS via the Microsoft Platform protocol. The purpose of the activity is to schedule a maintenance window for upcoming Windows updates, so that alarm events during that window can be appropriately attributed to the maintenance event.
+
+The content and screenshots for this tutorial were created in DataMiner version 10.4.2.
+
+<!-- Expected duration: TBD -->
+
+## Prerequisites
+
+- A DataMiner System using DataMiner 10.3.6 or higher, which is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud).
+- A Low-Code Apps license.
+- A user account with all [user-definable apps user permissions](xref:DataMiner_user_permissions#modules--user-definable-apps) enabled
+
+> [!NOTE]
+> If you use a [DaaS system](xref:Creating_a_DMS_in_the_cloud), these prerequisites are automatically met.
```

Comment:
@ThomasRich-Skyline, could you confirm that all three of these prerequisites are automatically met with DaaS, including the LCA license and the user permissions?